hpi-schul-cloud/schulcloud-server
# add tldraw application level metrics

# Description

QA note:
change tested on env:
https://bc-6059-tldraw-application-metrics.brb.dbildungscloud.dev/rooms/659d0c78bff8d8a6c77f330c/board
and grafana 
https://grafana.dbildungscloud.dev/d/b7dd7ec2-10a1-4971-9d8a-4d55581f4d7c/devops-tldraw?orgId=1&refresh=1m
The first 2 metrics are implemented:
how many Tldraw's boards are there; - no issues found
how many users are using the Tldraw's; - no issues found
the third:
what's the average number of users present in a Tldraw's board. - is excluded from the implementation 
more see - https://ticketsystem.dbildungscloud.de/browse/BC-6059



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-6059


## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
